### ISSUE TITLE
unique module ids in builds not great for projects

### ISSUE THREAD

**Issue by matthewmueller** [makojs/js#73]
When the module ids change, your files will always be different. This makes testing the output harder, diffs are more difficult to understand and the files are noisy in git if you check in your build files. 

Was there an issue with the browser pack ids?

```diff
-(function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({"0255cdfd-e26d-4376-ad42-c7583a067ca8":[function(require,module,exports){
+(function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({"c9c6c0e0-d7ac-4e03-b6f5-19fda424712d":[function(require,module,exports){
 module.exports = 'nested'

-},{}],"238e7cb0-8f70-4f67-8a11-c3af31864433":[function(require,module,exports){
+},{}],"cf18380d-f57e-4c50-b1a9-644a3937cfa4":[function(require,module,exports){
 module.exports = require('./nested')

-},{"./nested":"0255cdfd-e26d-4376-ad42-c7583a067ca8"}]},{},["238e7cb0-8f70-4f67-8a11-c3af31864433"])
+},{"./nested":"c9c6c0e0-d7ac-4e03-b6f5-19fda424712d"}]},{},["cf18380d-f57e-4c50-b1a9-644a3937cfa4"])
```

^ these have the same content, just their ids are different.

**Comment by dominicbarnes** [comment]
Ah, yeah that's definitely a problem. The main reason I opted for uuids instead was to prevent clobbering across different builds. I supposed I could use `file.initialPath` as the ID instead of the internal ID, that should be fine for the purposes of this module.

**Comment by dominicbarnes** [comment]
Ok, I released `v0.21.10` with this change, let me know if you have any trouble with it.
